src/dbspro/utils.py

```diff
@@ -6,10 +6,30 @@
 import os
 import re
 import subprocess
+import dnaio
+import pandas as pd
 
 logger = logging.getLogger(__name__)
 
 
+def get_abcs(abc_fasta_file):
+    """
+    Helper function to get ABC sequences and names into pandas dataframe
+    :param abc_fasta_file:
+    :return: dataframe:
+    """
+    with dnaio.open(abc_fasta_file, fileformat="fasta", mode="r") as abc_fasta:
+        abc = pd.DataFrame([{"Sequence": entry.sequence, "Target": entry.name} for entry in abc_fasta])
```

Comment:
If we instead add a `^` to normal fasta sequences here we a) follow fasta format, b) ensure it's impossible to not give anchored sequences and c) can remove rows 24-28.

Reply:
So the issue here is that the cutadapt requires a fasta file with the adaptors to demultiplex. Thus I cannot supply the sequence modified with the anchor '^' directly. What I could do however is to read in the fasta and make a copy of if that guarantees the anchor modification.